src/main/java/core/basesyntax/InvalidDataException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class InvalidDataException extends RuntimeException {
+    InvalidDataException(String message) {
```

Comment:
The constructor of the `InvalidDataException` class should be public to ensure that it can be accessed from outside the package.